frontend/pages/events/index.vue

```diff
@@ -20,10 +20,15 @@
         :event="event"
       />
     </div>
+    <div class="pt-3 pb-6 space-y-6 md:pt-4 w-1/2">
+      <CardDatePicker></CardDatePicker>
```

Comment:
Ah btw, with Vue it's totally ok to do a self closing component tag, so we just need `<CardDatePicker />` :)